app/Models/Category.php

```diff
@@ -100,7 +111,7 @@ public function feeds(): array {
 			foreach ($this->feeds as $feed) {
 				$this->nbFeeds++;
 				$this->nbNotRead += $feed->nbNotRead();
-				$this->hasFeedsWithError |= $feed->inError();
+				$this->hasFeedsWithError = $feed->inError();
```

Comment:
This change also looks wrong

Reply:
You can make `hasFeedsWithError` to a `bool|int` to avoid having to cast all the time and also keep performance